api/v1alpha1/fio_types.go

```diff
@@ -57,6 +61,11 @@ type FioSpec struct {
 	// PodConfig contains the configuration for the benchmark pod, including
 	// pod labels and scheduling policies (affinity, toleration, node selector...)
 	PodConfig PodConfigurationSpec `json:"podConfig,omitempty"`
```

Comment:
unrelated, but optional flag is missing here.

Reply:
resolved by ed1526156